kolibri/plugins/user/views.py

```diff
@@ -1,6 +1,14 @@
 from __future__ import absolute_import, print_function, unicode_literals
 from django.views.generic.base import TemplateView
+from kolibri.auth.models import Facility
 
 
 class UserView(TemplateView):
     template_name = "user/user.html"
+
+    def get_context_data(self, **kwargs):
+        context = super(UserView, self).get_context_data(**kwargs)
+
+        context['currentFacilityId'] = Facility.get_default_facility().id
```

Comment:
Is there no `kwargs["request"].user.facility` or something here? To minimize updates needed later when we upgrade with multi-facility support?